transitions/index.js

```diff
@@ -103,6 +117,83 @@ const processChange = (change, callback) => {
     });
 };
 
+const deleteInfoDoc = (change, callback) => {
+  db.medic.get(`${change.id}-info`, (err, doc) => {
+    if (err) {
+      if (err.statusCode === 404) {
+        // don't worry about deleting a non-existant doc
+        return callback();
+      }
+      return callback(err);
+    }
+    doc._deleted = true;
+    db.medic.insert(doc, callback);
+  });
+};
+
+const getAdminNames = callback => {
+  const options = { key: [ 'user-settings' ], include_docs: true };
+  db.medic.view('medic-client', 'doc_by_type', options, (err, results) => {
```

Comment:
I am really wondering if a new view would be better here (`admin_users`). They view will be very small, and it means that we don't have to load over every user document every time a delete happens.

Reply:
You're right - this could be a big deal for big databases. I'll fix it.